packages/resolver/lib/sources/fs.ts

```diff
@@ -65,18 +65,21 @@ export class FS implements ResolverSource {
     ];
 
     let body, filePath;
-    possiblePaths.forEach(possiblePath => {
+    for (const possiblePath of possiblePaths) {
       try {
         const resolvedSource = fs.readFileSync(possiblePath, {
           encoding: "utf8"
         });
         body = resolvedSource;
         filePath = possiblePath;
+        
+        return { body, filePath };
       } catch (error) {
         // do nothing
       }
-    });
-    return { body, filePath };
+    }
+    
+    throw new Error(`Path resolving failed for ${importPath}`);
```

Comment:
This might be fine, but based on the test case failing in the `PACKAGES` CI job, it appears there's already error handling for the fallback (which expects an empty object returned). You _could_ update the test case assertion (and possibly others that may need to be updated), but it would be cleaner to just stay with the current fallback code.